articles/tokens/index.md

```diff
@@ -41,11 +41,11 @@ When a user authenticates via Auth0 with another social provider's authenticatio
 
 [More information](/tokens/idp)
 
-## Auth0 `refresh_token`
+## Refresh Token
 
-The Refresh token is a long-lived token that is used to obtain a new [`id_token`](#auth0-idtoken-jwt) after a previous one has expired.  This is useful for applications running on mobile devices that call remote APIs and do not want to require the user to log in every time the user uses the mobile app.
+The Refresh Token, commonly referred to as `refresh_token`, is a long-lived [JWT](/jwt) that is used to obtain a new [id_token](#id-token) or a new [access token](#access-token), after a previous one has expired.
```

Comment:
Refresh tokens are not necessarily JWTs. Auth0 might return them as JWTs, but client applications should have no expectation about their format.